webapp/src/pages/PlayPage/PlayPage.tsx

```diff
@@ -1,42 +1,65 @@
+import './PlayPage.css';
 import * as React from 'react';
-import { ButtonVariant, IconVariant } from '../../shared/components';
+import {
+  ButtonProps,
+  ButtonVariant,
+  IconVariant,
+  Input,
+  InputVariant,
+  MediumAvatar,
+  NavigationBar,
+  ReactiveButton,
+  RowItem,
+  RowsList,
+} from '../../shared/components';
 import { useNavigate } from 'react-router-dom';
-import { MainTabTemplate } from '../../shared/components/index';
-import { SearchContextProvider } from '../../shared/context/SearchContext';
-import { PLAY_GAME_TRAIN_URL } from '../../shared/urls';
-import { ENTRIES_LIMIT } from '../../shared/constants';
-import { Query } from '../../shared/types';
+import {
+  AVATAR_EP_URL,
+  PLAY_GAME_TRAIN_URL,
+  PLAY_GAME_URL,
+  USER_ME_URL,
+} from '../../shared/urls';
 import socket from '../../shared/socket';
 import { useGamePairing } from '../../shared/hooks/UseGamePairing';
 import { gameQueueClientToServerWsEvents } from 'transcendence-shared';
-import { GamePairingStatusDtoGameQueueStatusEnum } from '../../shared/generated';
+import {
+  GamePairingStatusDtoGameQueueStatusEnum,
+  User,
+} from '../../shared/generated';
+import { Link } from 'react-router-dom';
+import { useAuth } from '../../shared/hooks/UseAuth';
+import { LandingPage } from '..';
+
+type GameWithPlayers = {
```

Comment:
We could share this data model somewhere in transcendence-shared/pong engine, couldn't we?

Reply:
I see. That's complicated. I imagine the packages cannot forward reference a type, or import it from transcendence-app. Let's forget about this for now. I'll investigate some time in the future, anyway.